app/assets/javascripts/ui_models/app_state/sync_state.ts

```diff
@@ -0,0 +1,36 @@
+import { SyncOpStatus } from "@standardnotes/snjs";
+import { action, makeObservable, observable } from "mobx";
+
+export class SyncState {
+  inProgress = false;
+  errorMessage?: string = undefined;
```

Comment:
`undefined` is used when there's an unintentional "mistake". If you want nullable values you should use `null` or not assign them anything (which will result in having `undefined` as a value) @baptiste-grob

Reply:
Most codebases have to at some point decide if they want to use `null` or `undefined` to signify "the absence of". In our case we chose `undefined` a while ago (most notably in SNJS) so this is just following that convention :)